test_runner/fixtures/pageserver/utils.py

```diff
@@ -198,7 +198,7 @@ def wait_for_last_record_lsn(
     lsn: Lsn,
 ) -> Lsn:
     """waits for pageserver to catch up to a certain lsn, returns the last observed lsn."""
-    for i in range(100):
+    for i in range(1000):
```

Comment:
Because in some CI runs debug version  of PS was not able to caught up in 10 seconds (100*0.1).
One of the suggestions was to add parameter (delay) to this method.
But I do not see much sense in it: if there is a bug and server is not able to caught up, then its is not critical that we spent 100 instead of 10 seconds before this error will be detected.

We may beed parameter if we do not consider such large caught up delay as something normal.
But in case of debug build and very specific testing configuration I do not think that we should try to check enforce some real-time conmtratints.

Reply:
> Because in some CI runs debug version of PS was not able to caught up in 10 seconds (100*0.1).

Hmm, why is that? We have a lot of existing tests that call `wait_for_last_record_lsn`, why does it take so long in this particular test?

Reply:
100k subxacts?